pkg/config/app/kuma-dp/testdata/default-config.golden.yaml

```diff
@@ -1,19 +1,20 @@
 controlPlane:
-  url: https://localhost:5678
+  caCert: ""
+  caCertFile: ""
   retry:
     backoff: 3s
     maxDuration: 5m0s
-  caCert: ""
-  caCertFile: ""
+  url: https://localhost:5678
 dataplane:
-  proxyType: dataplane
   drainTime: 30s
+  proxyType: dataplane
 dataplaneRuntime:
   binaryPath: envoy
+  resources: {}
```

Comment:
This is strange especially when the `omitempty` tag is present

Reply:
I don't know why, it happens, this is the result of golden files generation